### PR TITLE
Implementation of Bridge and Torch Problem in Python

### LINKED ISSUE #1768: Add "Bridge and torch problem" Implementation
Bridge and torch problem is a popular puzzle where four people need to cross a faulty bridge in minimum time with only two people able to cross at a time. They have one torch and every time a pair crosses the bridge, one needs to come back. Given the time taken by the four people to cross the bridge, we have to compute the minimum time taken to cross the bridge. I would like to be assigned this problem @singh-shreya6

### PR DESCRIPTION
**Pull request by HarshCasper**
Fixes #1768 

#### Checklist:
- [X] 4 space indentation.
- [X] Coding conventions are followed.
- [X] Input is taken dynamically.
- [X] Sample Input / Output is added at the end of file.
- [X] Logic Documentation (Comments).
- [X] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
- Added Bridge and Torch Problem

#### Languages Used:
- Python

#### Files Added:
- bridge-torch.py

Thanks!

### CODE REVIEW

**Comment by HarshCasper** [comment]
Kindly check it out @jainaman224 @singh-shreya6 @SakshiShreya

File: Bridge and Torch/bridge-torch.py [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+have one torch and every time a pair crosses the bridge, one needs to come back.
+Given the time taken by the four people to cross the bridge, we have to compute 
+the minimum time taken to cross the bridge.
+"""
+
+def bridgeTorch(arr):
+    # Sorting all the Inputs
+    arr.sort()                                  
+    if len(arr)>3:                               
+        a = arr[0]+arr[-1]+min(2*arr[1],arr[0]+arr[-2]) #Calculating the Minimum Time
```
Comment (raghav-dalmia) [inline Bridge and Torch/bridge-torch.py:14]: add more comment to clarify your logic

File: Bridge and Torch/bridge-torch.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+        return  a + bridgeTorch(arr[:-2]) #Recursion on Remaining Inputs                 
+    else:
+        return sum(arr[len(arr)==2:]) #If Length is less than 3, just add all the inputs
```
Comment (raghav-dalmia) [inline Bridge and Torch/bridge-torch.py:17]: just use `sum(arr)` as have already used if condition. Try to make your code as simple as possible.

**Comment by raghav-dalmia** [comment]
make sure to squash your commits after these changes

**Comment by HarshCasper** [comment]
I will add all the changes needed by Sunday.

**Comment by somya-kapoor** [comment]
Are you working on this?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days I am closing this PR.

**Pull request closed without merging**